phpBB/styles/prosilver/theme/mentions.css

```diff
@@ -0,0 +1,88 @@
+/* -------------------------------------------------------------- /*
+    $Mentions
+/* -------------------------------------------------------------- */
+
+/* stylelint-disable selector-max-compound-selectors */
+/* stylelint-disable selector-no-qualifying-type */
+
+/* Mention block
+---------------------------------------- */
+.mention {
+	font-weight: bold;
+}
+
+/* Mention dropdown
+---------------------------------------- */
+.mention-container {
+	text-align: left;
+	border-radius: 2px;
+	position: absolute;
+	z-index: 999;
+	overflow: auto; /* placed here for list to scroll with arrow key press */
+	max-height: 200px;
+	transition: all 0.2s ease;
+}
+
+.mention-list {
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+}
+
+.mention-media {
+	display: inline-flex;
+	flex-shrink: 0;
+	justify-content: center;
+	align-items: center;
+	margin-right: 8px;
+	margin-left: 0;
+}
+
+.mention-media-avatar {
```

Comment:
given the changes to the html I think this should be merged into the class above